src/applications/vaos/manifest.json

```diff
@@ -2,6 +2,6 @@
   "appName": "VA Online Scheduling",
   "entryFile": "./vaos-entry",
   "entryName": "vaos",
-  "rootUrl": "/health-care/schedule-view-va-appointments/appointments",
+  "rootUrl": "/health-care/schedule-view-va-appointments",
```

Comment:
This change here dictates which page to go to after successful login and when access from other applications. Somewhere in the build process this 'rootUrl' is used to update all the links through out vet-website. You can rebuild your content locally using 'yarn build'.